behat.yml.dist

```diff
@@ -15,12 +15,14 @@ default:
         - SendInterviewContext
         - EventContext
         - TalkContext
+        - SpeakerContext
   extensions:
     Behat\Symfony2Extension:
       kernel:
         bootstrap: features/bootstrap/bootstrap.php
         class: App\Kernel
     Behat\MinkExtension:
+        files_path: "http://localhost:8080/public/images/"
```

Comment:
As we create a make command to launch the behat tests in the php container, this url won't work as there's no web server on that container (the php one). You must use the same url as the `base_url` param.

Reply:
I dont have this problem with my php container. But I change this path.

Reply:
`files_path` is expecting a relative or absolute filesystem path (and not an url) as you can see in the [doc](https://github.com/Behat/MinkExtension/blob/master/doc/index.rst#additional-parameters).

As you passed an url, when running your tests, the return value of the function `realpath` in https://github.com/Behat/MinkExtension/blob/master/src/Behat/MinkExtension/Context/MinkContext.php#L212 is `/default_speaker.svg` for the first 2 tests, and nothing for the last.